gsuite_driver/settings.py

```diff
@@ -5,6 +5,8 @@
 :mod:`gsuite-driver.settings` -- GSuite Driver Configuration
 * Environment variables used
 * GSUITE_DRIVER_PREFIX
+* GSUITE_DRIVER_ARCHIVING
```

Comment:
due to https://github.com/andrewkrug/gsuite-community-drive-driver/blob/abf9e53aaeccf78864eec1ce45708109042f6767/gsuite_driver/main.py#L34
shouldnt this be:
`GSUITE_DRIVER_ARCHIVE` ?